examples/scripts/utils/downstream/cleanup_downstream_resources.py

```diff
@@ -0,0 +1,68 @@
+"""
+Example script for checking and cleaning up downstream datasets.
+NOTE: USE AT YOUR OWN RISK. Deleting datasets can sometimes break the system
+and should only be done when necessary. Always take backup before you perform the cleanup.
+"""
+from tamr_toolbox import utils
```

Comment:
Please refactor to use `import tamr_toolbox as tbox` instead. This is a new addition to our style guide for example scripts/snippets